pkg/apis/workspaces/v1alpha2/endpoint.go

```diff
@@ -92,7 +92,7 @@ type Endpoint struct {
 	Protocol EndpointProtocol `json:"protocol,omitempty"`
 
 	// Describes whether the endpoint should be secured and protected by some
-	// authentication process
+	// authentication process. This requires a protocol of "https" or "wss".
```

Comment:
```suggestion
	// authentication process. This requires a protocol of `https` or `wss`.
```

We have implemented minimal markdown support for the Json schema descriptions generated from the GO comments. That's also used in the generated API reference.
So using backticks here will allow a nicer markdown rendering, like it is already done [here](https://github.com/devfile/api/pull/268/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json#diff-075e68dfa603f8a68957e3f7e1d69ab13eeab7363188e5c6e9f7a13a6d6070fcR87) for example.